UI/templates/welcome.html

```diff
@@ -21,16 +25,18 @@
     <div class="welcome">
         <div class="welcome-message">
             <h1>Welcome to Ride-my-way</h3>
-            <h5>This is a carpooling application that provides drivers 
-                with the ability to create ride offers and passengers to join available ride offers.</h5>
-            <hr>   
+                <h5> Sharing the costs of the ride with the driver makes your trip really affordable.</h5>
+                <hr>
+                <p>Get a free account
+                    <a href="registration.html">here</a>!</p>
         </div>
-            <p>Get a free account <a href="registration.html">here</a>!</p>
+
     </div>
     <!--footer fixed-->
     <div class="footer">
         <p>Ride-my-way &copy; 2018</p>
     </div>
-    
+
 </body>
+
 </html>
```

Comment:
see comment above

Reply:
I have noticed where I went wrong. I overlooked the optional feature "Users can only see and respond to ride offers from their friends on the application", Thought it was nice for friends to comment on their friends ride status. That is why I introduced friends.